pkg/webhook/mutation/pod/webhook.go

```diff
@@ -166,24 +164,26 @@ func (wh *webhook) handlePodMutation(ctx context.Context, mutationRequest *dtweb
 			continue
 		}
 
-		if err := mutator.Mutate(ctx, mutationRequest); err != nil {
-			return err
+		resultMutationRequest := mutationRequest.DeepCopy()
+		if err := mutator.Mutate(ctx, resultMutationRequest); err != nil {
+			log.Error(err, "pod mutator failed", "pod", mutationRequest.Pod.Name)
```

Comment:
you could collect the errors instead of logging each of them, like [here](https://github.com/Dynatrace/dynatrace-operator/blob/1fad0e42ee963470471b8e9bb556473679c5c725/pkg/controllers/dynakube/controller.go#L388)